aiida/orm/nodes/process/calculation/calcjob.py

```diff
@@ -445,11 +445,11 @@ def get_last_job_info(self):
         """
         from aiida.schedulers.datastructures import JobInfo
 
-        last_job_info_serialized = self.get_attribute(self.SCHEDULER_LAST_JOB_INFO_KEY, None)
+        last_job_info_dictserialized = self.get_attribute(self.SCHEDULER_LAST_JOB_INFO_KEY, None)
 
-        if last_job_info_serialized is not None:
+        if last_job_info_dictserialized is not None:
             job_info = JobInfo()
-            job_info.load_from_serialized(last_job_info_serialized)
+            job_info.load_from_dict(last_job_info_dictserialized)
```

Comment:
This can become `job_info = JobInfo.load_from_dict(last_job_info_dict_serialized)`

Reply:
Done